R/check-logic.R

```diff
@@ -67,15 +67,16 @@
 #' @name check_logic
 #' @export
 check_correct <- function(state, ...) {
+  force_diagnose <- tw$get("state")$get('force_diagnose')
```

Comment:
I think the `state` argument should have force_diagnose on it (due to the change in `state.R`). I would get force_diagnose from there (AFAIK contemporary testwhat code shouldn't need to use tw$get in the SCTs)

Reply:
I think I wrote it that way first, but some tests failed because it didn't have the attribute.
Maybe because of a child state being manually constructed instead of copied and adjusted?
Good to know that in general it should be used!

Reply:
The tests fail on this way of writing:
https://github.com/datacamp/testwhat/blob/master/tests/testthat/test-check-logic.R#L135

Reply:
I resolved it using the pointer you gave me!